src/Confluent.Kafka/Loggers.cs

```diff
@@ -21,8 +21,14 @@
 
 namespace Confluent.Kafka
 {
+    /// <summary>
+    ///     OnLog callback event handler implementations.
+    /// </summary>
     public static class Loggers
     {
+        /// <summary>
+        ///     The method used to log messages by default.
+        /// </summary>
         public static void ConsoleLogger(object obj, LogMessage logInfo)
         {
             var now = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss.fff");
```

Comment:
though TBH locale specific date times annoy me and the one used here is neutral. that said, open to changing it. not bothering with a github issue.